bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -84,6 +87,68 @@ determine_chart_version() {
         fail || return
 }
 
+assert_not_empty_list() {
+    run "$@"
+    assert_success || return
+    run jq_output length
+    assert_success || return
+    refute_output 0 || return
+}
+
+assert_true() {
+    run --separate-stderr "$@"
+    assert_success || return
+    is_true "$output" || return
```

Comment:
Is this maybe a bit too forgiving? It will treat anything that is not one of `'', '0', 'no', 'false'` as `true`. So if for some reason the status field could say `offline`, it would be treated as true.